portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,111 @@
 /**
  * Adds a random greeting to the page.
```

Comment:
This comment seems to have been detached from it's function, and also needs updating.

Reply:
fixed